src/domains/shared-app/api/authorizer-config/v1/_base_policy.xml

```diff
@@ -1,80 +1,7 @@
 <policies>
   <inbound>
     <base />
-
```

Comment:
can you explain why you removed these lines of code?

Reply:
in a previous commit, I copied the wrong _base_policy.xml_ file for the new APIs so in this commit I deleted the unused lines in this files.